### PR TITLE
feat: Introduce new notice-file-generation

### PR DESCRIPTION
**Pull request by toninis**
## Summary

Introduce new packages on NOTICE.txt file 
Related PR: https://github.com/mattermost/delivery-platform/pull/121 

## Ticket Link

https://mattermost.atlassian.net/browse/CLD-6877

### CODE REVIEW

**Review by amyblais — APPROVED** [review]

File: .config/notice-file/config.yaml [hunk 1]
```diff
@@ -1,12 +1,10 @@
 ---
-
 title: "Mattermost Calls"
 copyright: "©2015-present Mattermost,Inc. All Rights Reserved. See LICENSE for license information."
 description: "This document includes a list of open source components used in Mattermost Calls plugin, including those that have been modified."
 search:
   - "go.mod"
   - "build/go.mod"
+  - "webapp/package.json"
```
Comment (streamer45) [inline .config/notice-file/config.yaml:8]: Technically speaking we also have `standalone/package.json` which mostly overlaps with `webapp/package.json` but it's a different thing. We also have `server/public/go.mod` and `lt/go.mod`.

Comment (toninis) [reply .config/notice-file/config.yaml:8]: We mainly need the ones that are actually used on production . Which files depict those dependencies ?

Comment (streamer45) [reply .config/notice-file/config.yaml:8]: All of those, except `lt/go.mod` would be part of our builds.

Comment (streamer45) [reply .config/notice-file/config.yaml:8]: From the ones there already, `build/go.mod` wouldn't really be part of our releases. It's just a dev dependency.

**Review by streamer45 — APPROVED** [review]
Thanks!

**Pull request merged**